modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestForm/steps/Review/components/Body/components/GoalsEntries/GoalsEntries.tsx

```diff
@@ -55,6 +55,9 @@ const GoalsEntries = ({mdfRequest}: IProps) => (
 				{
 					columnKey: 'value',
 					label: '',
+					render: (data: any) => (
+						<p style={{whiteSpace: 'pre-wrap'}}>{data}</p>
+					),
```

Comment:
We need to avoid to use any type and style prop.

Check if some bootstrap class can attend you.
What this data is? A Element?

Reply:
Fixed. Could be a string, an array of strings or undefined.